drivers/builtin/include/mbedtls/platform.h

```diff
@@ -446,6 +446,32 @@ mbedtls_platform_context;
 #include "platform_alt.h"
 #endif /* !MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT */
 
+#if !defined(MBEDTLS_ENTROPY_HARDWARE_POLL_MIN)
+#define MBEDTLS_ENTROPY_HARDWARE_POLL_MIN     32     /**< Minimum for the hardware source */
+#endif
+
+#if defined(MBEDTLS_ENTROPY_HARDWARE_POLL)
+/**
+ * \brief           User defined callback function that is used from the entropy
+ *                  module to gather entropy data from some hardware device.
+ *
+ * \warning         This is not provided by Mbed TLS.
+ *
+ * \param[out] output           Output buffer where the entropy data will be
+ *                              stored.
+ * \param[in] output_size       Size of the \p output buffer in bytes.
+ * \param[out] output_len       Amount of bytes effectively written in the
+ *                              \p output buffer.
+ * \param[out] entropy_content  Measure of the entropy content (in bits) of the
+ *                              data written in the \p output buffer. The pointer
+ *                              can be \c NULL in case the caller is not interested
+ *                              in such information, so the function must handle
+ *                              this.
```

Comment:
We don't need to support callers that don't want the value. `entropy_content` is guaranteed to be non-null.